docs/self-managed/components/orchestration-cluster/operate/operate-configuration.md

```diff
@@ -26,223 +26,12 @@ See the [core settings documentation](/self-managed/components/orchestration-clu
-  zeebe:
-    # Gateway host and port
-    gatewayAddress: localhost:26500
-```
-
-### Intra-cluster secure connection
-
-You can enable intra-cluster TLS secured connections between Operate and Zeebe by applying the following configuration properties.
-
-| Name                                                | Description                                                                                                         | Example value                     |
-| :-------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------ | :-------------------------------- |
-| zeebe.gateway.cluster.initialContactPoints          | Zeebe Gateway initial contact points.                                                                               | [gateway-0:26502,gateway-1:26502] |
-| zeebe.gateway.cluster.security.enabled              | Connection should be secure via Transport Layer Security (TLS).                                                     | true                              |
-| zeebe.gateway.cluster.security.certificateChainPath | Path to certificate used by Zeebe. This is necessary when the certificate isn't registered in the operating system. | /path/to/cert.pem                 |
-| zeebe.gateway.cluster.security.privateKeyPath       | Path to certificate's key used by Zeebe.                                                                            | /path/to/private.key              |
-| zeebe.gateway.cluster.advertisedHost                | Advertised hostname in the cluster.                                                                                 | operate                           |
-| zeebe.gateway.cluster.memberId                      | Member ID for the cluster.                                                                                          | operate                           |
-
-For extended configuration and guidelines, refer to [secure cluster communication](../zeebe/security/secure-cluster-communication.md) and [gateway configuration](../zeebe/configuration/gateway.md).
+See the [secondary storage documentation](/self-managed/components/orchestration-cluster/core-settings/configuration/properties.md#secondary-storage).
 
 ## Zeebe Elasticsearch or OpenSearch exporter
 
 :::note
-Please refer to [Supported Environments](../../../../reference/supported-environments.md#camunda-8-self-managed) to find out which versions of Elasticsearch or OpenSearch are supported in a Camunda 8 Self-Managed setup.
+**Version 8.8:** The importer is required only for brownfield deployments (upgrades from 8.7). Keep it enabled until all remaining 8.7 records have been imported. After that, the [Camunda Exporter](/self-managed/components/orchestration-cluster/zeebe/exporters/camunda-exporter.md) takes over and writes directly to the Operate indices. The importer can then be safely disabled for subsequent restarts or deployments.
```

Comment:
personal opinion, I don't think we should label migration scenarios as "brownfield". While we often use that term internally, I don't think it is intuitive enough for our users to understand what we are referring to. I would prefer something like "migration scenarios"